omnichain/swap/tasks/interact.ts

```diff
@@ -0,0 +1,36 @@
+import { task } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { parseEther } from "@ethersproject/units";
+import { getAddress } from "@zetachain/protocol-contracts";
+import { prepareData, trackCCTX } from "@zetachain/toolkit/helpers";
+import { BigNumber } from "@ethersproject/bignumber";
+
+const main = async (args: any, hre: HardhatRuntimeEnvironment) => {
+  const [signer] = await hre.ethers.getSigners();
+  console.log(`🔑 Using account: ${signer.address}\n`);
+
+  const targetZRC20 = getAddress("zrc20" as any, args.destination as any);
```

Comment:
You're right, we don't need `any` for `zrc20`, but we need to cast to `any` for the network name, because `getAddress` expects the second argument to be of type `ZetaProtocolNetwork` and. I think importing types just for arguments is a bit of an overkill.